.github/workflows/test_php.yml

```diff
@@ -22,37 +32,45 @@ jobs:
             version: "8.1.14"
             version-short: "8.1"
             command: composer test \&\& composer test_c
+            presubmit: true
```

Comment:
We should keep this consistent across test files and use `continuous-only`

Reply:
Definitely agreed. Not sure how I missed this file, but it's now updated to `continuous-only`.